WebRoot/css/zm.css

```diff
@@ -1982,14 +1982,49 @@ div.CompDragArea * {
 	padding:5px;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+.ZmTaskMultiView .newTaskBanner .ImgAdd-svg {
+	@TaskListViewAddNewTaskIcon@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble {
+	@TaskReadingPaneRight-ListViewItem@
+}
+
+.ZmTaskMultiView .DwtListView .RowDouble td {
+	@TaskReadingPaneRight-ListViewItemCheckboxCell@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskSubject {
+	@TaskReadingPaneRight-ListViewItemSubject@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskDate {
+	@TaskReadingPaneRight-ListViewItemDate@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow {
+	@TaskReadingPaneRight-ListViewItemBottomContainer@
 }
 
-.newTaskBannerSep {
-	border-bottom:2px solid #AAA;
+.ZmTaskMultiView .DwtListView .BottomRow .IconOuterWrapper {
+	@TaskReadingPaneRight-ListViewItemBottomIconOuterWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconInnerWrapper {
+	@TaskReadingPaneRight-ListViewItemBottomIconInnerWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-Rows .RowDouble .ImgCheckboxUnchecked-svg,
+.ZmTaskMultiView .DwtListView .DwtListView-Rows .RowDouble .ImgCheckboxChecked-svg {
+	@TaskReadingPaneRight-ListViewItemCheckIcon@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble > table {
+	@FullHeight@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble > table > tbody > tr > td:last-of-type > table:first-of-type  {
+	height:60%;
```

Comment:
yes, we intend to target top table, here.
It will take 60% height of containing cell.